### PR TITLE
Add Why Aptos section to top of Community index, link to Genesis Medi…

### PR DESCRIPTION
**Pull request by clay-aptos**
…um post

### CODE REVIEW

File: developer-docs-site/docs/community/index.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ We welcome your own [contributions](https://github.com/aptos-labs/aptos-core/blo
 
 As always, adhere to the [Aptos Code of Conduct](https://github.com/aptos-labs/aptos-core/blob/main/CODE_OF_CONDUCT.md) when taking part in our ecosystem.
 
+## Why Aptos?
```
Comment (davidiw) [inline developer-docs-site/docs/community/index.md:12]: Have you considered pulling more from here: https://aptosfoundation.org/currents/why-aptos-8-innovations-powering-aptos-network

That way we can kill the link to the Labs medium, not talk at all about the core team, and focus on the tech and the ecosystem.

Comment (davidiw) [reply developer-docs-site/docs/community/index.md:12]: btw why is this here versus say the main page?

Comment (clay-aptos) [reply developer-docs-site/docs/community/index.md:12]: > btw why is this here versus say the main page?

We have gotten feedback from reviewers our main page is already overwhelming. Leading with another paragraph or two seemed to be going against those wishes.

Comment (clay-aptos) [reply developer-docs-site/docs/community/index.md:12]: @davidiw , another option is to create a distinct Why Aptos? page in the Start menu, perhaps as the first page above See what's new, that summarizes the features from the governance page.

And still another option is to rename in navigation and expand the white paper introduction to become the Why Aptos? page:
https://aptos.dev/aptos-white-paper/aptos-white-paper-index

It already offers a features list and is among the most popular pages on the site. What do you think? Thanks!

Comment (clay-aptos) [reply developer-docs-site/docs/community/index.md:14]: After getting guidance on messaging, I have removed references to Core and linked to the recommended Foundation page. If we want to pull that content in still, perhaps that separate page with a merge of the white paper intro would work. @davidiw , please take another look.

**Comment by davidiw** [comment]
This doesn't address the feedback and the intended content would serve better on the main page. Let's reconsider some alternative.

**Pull request closed without merging**